presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1271,10 +1272,10 @@ private DataTypeTest hstoreTestCases(DataType<Map<String, String>> varcharMapDat
     public void testUuid()
```

Comment:
I would make these tests parametric instead. IDE supports to run test with particular parameters which is nice. That it is also cleaner, separate tests are in separate test executions.